README.md

```diff
@@ -41,7 +41,7 @@ $ git clone https://github.com/rero/rero-ils-ui.git/
 $ cd rero-ils-ui
 $ nvm use lts/dubnium
 # Install angular-cli
-$ npm i -g angular-cli
+$ npm i -g @angular/cli@8.3.25
```

Comment:
is it necessary ?

Reply:
In documentation, I think that yes.
I follow it and fetch angular-cli 1.0.0beta32 (or similar) which was a problem to have a development environment.